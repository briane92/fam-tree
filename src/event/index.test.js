/**
 * Created by beggl on 10/24/2017.
 */
import React from 'react'
import renderer from 'react-test-renderer'
import {Event} from './index'
import {shallow, mount, render} from 'enzyme'



it('Event component renders correct', ()=> {
    let e = {
        title: "Christmas of 1992",
        date: new Date(1992, 11, 25),
        description: "Christmas of 1992 was spent in New York City",
    }

    const component = renderer.create(<Event e={e}/>)
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
})

describe('<Event>t', function () {
    it('it renders the event with correct data',()=> {
        const event = {
            title: "Christmas of 1992",
            date: new Date(1992, 11, 25),
            description: "Christmas of 1992 was spent in New York City",
        }
        const wrapper = shallow(<Event e={event}/>)
        expect(wrapper.text()).toContain('Christmas of 1992')

    })

})


