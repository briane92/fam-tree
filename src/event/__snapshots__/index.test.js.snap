// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Event component renders correct 1`] = `
<div>
  <h1 />
   - 
  <h2 />
  <p />
</div>
`;
